sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/VisualStudioCoverageXmlReportParserTest.java

```diff
@@ -38,7 +43,14 @@
 
   @Rule
   public ExpectedException thrown = ExpectedException.none();
-  private Predicate<String> alwaysTrue = s -> true;
+  private FileService alwaysTrue;
```

Comment:
because in the case of VS coverage, when the `isAbsolutePath` is false, it will call `getFileByRelativePath`.
For the others, the expectation is not to call this.
I clarified these assumptions in the tests.